debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerQueryResultProcessor.java

```diff
@@ -35,10 +38,11 @@
  */
 class LogMinerQueryResultProcessor {
 
-    private final ChangeEventSource.ChangeEventSourceContext context;
+    private final ChangeEventSourceContext context;
     private final LogMinerMetrics metrics;
     private final TransactionalBuffer transactionalBuffer;
-    private final SimpleDmlParser dmlParser;
+    private final SimpleDmlParser legacyDmlParser;
```

Comment:
Couldn't there be an interface for the two implementations, and there'd only be a single field here, of this type?